.github/workflows/translations.yml

```diff
@@ -1,66 +1,87 @@
-name: Update translation source
+name: Update Translation Source
 
 on:
   workflow_dispatch:
   schedule:
-    # runs once per month
+    # runs at the start of each month (UTC)
```

Comment:
Is once a month enough?

Reply:
yes, we already discussed this

Reply:
Now, with the PR creation instead of a push to master (which is what we planned first, but did not end up working), I'm ok with something like 2 weeks. 👍 

We can always trigger a manual run on demand with one click though. That's what the "dispatch" is enabling.

Reply:
see https://github.com/Cockatrice/Cockatrice/pull/4543#issuecomment-1019732098